README.md

```diff
@@ -13,7 +13,7 @@ Install from PyPI
 
 or alternatively install development version directly from github
 
-    pip install -e git+git://github.com/farin/python-fakturoid-v3#egg=fakturoid
+    pip install -e git+git://github.com/jarovo/python-fakturoid-v3#egg=fakturoid-v3
```

Comment:
Are you intentionally mentioning a different GitHub repository...?

Reply:
Oh sorry. I made this PR as a result of a mistake.